products/idn/tools/cli/sanitize.md

```diff
@@ -0,0 +1,58 @@
+---
+id: cli-sanitize
+title: Sanitize
+pagination_label: CLI-Sanitize
+sidebar_label: Sanitize
+sidebar_position: 2
+sidebar_class_name: cli
+keywords: ['cli', 'sanitize']
+description: Learn how to use the CLI to sanitize your har files in this guide.
+slug: /tools/cli/sanitize
+tags: ['CLI']
+---
+
+
+## Sanitize
+
+Learn how to use the CLI to sanitize your har files in this guide.
+
+In IdentityNow (IDN), you occasionally need to troubleshoot issues, and often some of the most helpful data can be a har file collected of your browsers api calls made while replicating the issue.
+
+The problem lies in the amount of information, specifically sensitive information that that file contains, due to the nature of what it is capturing.  This is where the CLI comes in.  The CLI can be used to sanitize the har file, removing sensitive information, while still leaving the file in a useful state for support to troubleshoot the issue.
+
+The existing support guide for collecting and sanitizing the har file [can be found here](https://support.sailpoint.com/csm?id=kb_article_view&sysparm_article=KB0011711#_gl=1*17esz5f*_gcl_au*MTczOTU0NDU4NC4xNjkxNjc5NzY2)
+
+- [Sanitize](#sanitize)
+  - [Command](#command)
+
+### Functionality
+
+The Sanitize command sanitizes a har file of sensitive data.
+Specifically it removes access tokens and origin urls similar to the examples below:
```

Comment:
Call out that it only removes IdentityNow access tokens and origin URLs.  It will not attempt to remove any other potentially sensitive information.

Reply:
Done